src/mrpro/operators/IdentityOp.py

```diff
@@ -42,3 +43,25 @@ def adjoint(self, x: torch.Tensor) -> tuple[torch.Tensor]:
             the input tensor
         """
         return (x,)
+
+
+class MultiIdentityOp(EndomorphOperator):
```

Comment:
Is this not covered in MultiIdentityOp.py?

Reply:
Ups. mistake in rebasing.